### PR TITLE
update ruby version

### PR DESCRIPTION
**Pull request by lucianosousa**

### CODE REVIEW

**Comment by lucianosousa** [comment]
@tauil

**Comment by lucianosousa** [comment]
@tauil what do u think about it?

File: app/helpers/proposals_helper.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def proposal_user(proposal)
   end
 
   def vote_box(event, proposal, user)
-    if user && user.has_vote_for?(proposal)
+    if user&.has_vote_for?(proposal)
```
Comment (tauil) [inline app/helpers/proposals_helper.rb]: Nice way to validate the new ruby version! hehe

**Comment by tauil** [comment]
:+1:

**Pull request merged**